internal/cmd/api-key/command_use.go

```diff
@@ -21,9 +21,9 @@ func (c *command) newUseCommand() *cobra.Command {
 		RunE:              c.use,
 	}
 
-	cmd.Flags().String(resourceFlagName, "", "The resource ID.")
+	c.addResourceFlag(cmd)
```

Comment:
It seems like this could end up suggesting invalid values for this flag (like sr clusters) which will result in `functionality not yet available for non-Kafka cluster resources`.

Is this something we want to prevent, or is it even worth preventing?

Reply:
I'm planning on dropping this flag altogether soon, since stored API keys are already associated with a Kafka cluster (and hopefully other cluster types in the future) and the user shouldn't need to supply it. For the time-being, I think the autocompletion is more helpful than it is unhelpful, so I'd like to keep it!

Reply:
Sounds good to me.